Update README. (#45)
googleinterns/step236-2020: README.md

# Firebase setup

> For this project, you need an existing firebase project and a domain with gSuite for business.

Reviewer: nit: the proper spelling of gSuite is G Suite. Also, I don't think "for business" is needed. The G Suite subscription that we have is G Suite Basic.

```suggestion
For this project, you need an existing Firebase project and a domain with G Suite.
```

Install the Firebase CLI.

```bash
npm install -g firebase-tools
```

### `firebase login`

Use it when you want to access the firebase for the first time. Later, you will be logged in automatically. Log in with an account assiociated with the project you are developing.

### `firebase init`

> Initialise new project on Firebase. Currently, firebase features that this project uses are: hosting, firestore and cloud functions.

Reviewer: nit: capitalization of the product names
```suggestion
Initialise new project on Firebase. Currently, Firebase features that this project uses are: Hosting, Firestore and Cloud Functions.
```

> Don't overwrite any configuration files when prompted by firebase init. We already configured these files for you.

Reviewer: nit: backtick-quote the command
```suggestion
Don't overwrite any configuration files when prompted by `firebase init`. We already configured these files for you.
```
